DesignSafe-CI/portal
# task/DES-1963 - Curation Directory mentions Categories for Other

## Overview: ##
Other-type projects don't have user-definied categories, so the tutorial explaining them was removed from other-type projects and replaced with a tutorial to move files.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1963](https://jira.tacc.utexas.edu/browse/DES-1963)

## Summary of Changes: ##
The template checks if the project's category is other, and displays the correct tutorial as needed.

## Testing Steps: ##
1. Create an empty other-type project or DM Frank to be added to one.
2. Check that the "Working directory" and "Curation directory" both display the tutorial for help moving files, and do not display the tutorial for creating categories
3. Create any other type of empty project
4. Check that the "Curation directory" displays the tutorial for creating categories and that the "Working directory" displays the tutorial for help moving files

## UI Photos:
Curation Directory for an empty other-type project:
 ![Screen Shot 2021-06-02 at 4 20 40 PM](https://user-images.githubusercontent.com/7462330/120553805-8fe0a980-c3be-11eb-8bde-a3f81c330874.png)


## Notes: ##
